### PR TITLE
Add GitHub workflow and use default Prettier settings

### PR DESCRIPTION
**Pull request by vilhelmprytz**

### CODE REVIEW

**Comment by vilhelmprytz** [comment]
It fails because the app is asking for the token in TTY. Is it possible to remove this feature entirely? :slightly_smiling_face:

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm i --production --unsafe-perm
+      # - run: npm test add when tests are written
+      - run: npx prettier --check "src/**/*.ts"
```
Comment (Linux123123) [inline .github/workflows/node.js.yml]: Actually we not only want to use `prettier`, but also `eslint` (I have set up the config in a way that it shows both `eslint` errors and `prettier` errors).
```suggestion
      - run: npm lint
```

File: .github/workflows/node.js.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        node-version: [14.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm i --production --unsafe-perm
```
Comment (Linux123123) [inline .github/workflows/node.js.yml]: This is not going to install dev dependencies. So the option `--production` isn't needed. Also the `--unsafe-perm` is only for docker images. Also `npm ci` might be better option, but I am not too sure about it.
```suggestion
      - run: npm ci
```

**Review by Linux123123 — CHANGES_REQUESTED** [review]

**Review by Linux123123 — APPROVED** [review]

**Comment by Linux123123** [comment]
LGTM

**Pull request merged**